### ISSUE TITLE
Document `Skylight::Normalizers::Normalizer` -> `Skylight::Core::Normalizers::Normalizer`

### ISSUE THREAD

**Issue by hmcfletch** [skylightio/skylight-ruby#120]
I'm not sure which release this happened in, but could you put something in the Changelog for this? It will break all third party normalizers.

**Comment by hmcfletch** [comment]
Agreed. Docs and examples would help a lot for those needing to make their own normalizers or those hooking into AS::Notifications. In the mean time, I think I mention in the Changelog for the right version would be helpful for those who run into this issue.

**Comment by wagenet** [comment]
By the way, `Skylight::Core` was introduced in 2.0.
